### ISSUE TITLE
Kotlin classes not recognised

### ISSUE THREAD

**Issue by gioPucela** [koral--/gradle-pitest-plugin#15]
When running pitest from the command line, it seems that no mutations are generated for Kotlin classes.

Kotlin classes are simply excluded somehow. This is at least the case when the test targeted classes as well as the test classes are written in Kotlin. However, the mutation and test reports are generated for the java classes.

My configuration is Android Studio 3.0 Beta 5, with kotlin_version = '1.1.51' and 'pl.droidsonroids.gradle:gradle-pitest-plugin:0.0.11'

**Comment by koral--** [comment]
Plugin is not yet compatible with 3.x.x beta Android Gradle plugin. I'll fix it ASAP.
Workaround for now is to add missing mutable code paths maunally, like this:
```
pitest {
    additionalMutableCodePaths = [project.file("build/tmp/kotlin-classes/<variant>")]
...
```
Where `variant` is a build variant e.g. `debug` or `release` when there are no product flavors.

**Comment by gioPucela** [comment]
Adding
```
pitest {
    additionalMutableCodePaths = [project.file("build/tmp/kotlin-classes/<variant>")]
...
```
worked! Thanks!

**Comment by koral--** [comment]
Fixed in version `0.1.2`.
